app/models/gsa18f/procurement.rb

```diff
@@ -76,7 +80,13 @@ def name
     end
 
     def self.approver_email
-      ENV['GSA18F_APPROVER_EMAIL'] || '18fapprover@gsa.gov'
+      raise 'No 18F approver was set' unless ENV['GSA18F_APPROVER_EMAIL']
+      ENV['GSA18F_APPROVER_EMAIL']
```

Comment:
This would set the value to nil if it isn't set in the env vars. Is this what we want?